.github/workflows/node.js.yml

```diff
@@ -45,13 +48,16 @@ jobs:
       - name: Repository checkout
         uses: actions/checkout@v4
 
+      - name: Enable Corepack
+        run: corepack enable
+
       - name: Node.js setup
         uses: actions/setup-node@v4
         with:
           node-version-file: .tool-versions
 
       - name: 📦 Install dependencies
-        run: yarn install --immutable --immutable-cache
+        run: yarn install
```

Comment:
I was going to recommend keeping `--immutable`, but `yarn` v4 is smart and defaults `--immutable` to true in CI.

Reply:
Is there something you are wanting me to do before you approve this?